frontend/src/main/webapp/index.html

```diff
@@ -6,6 +6,9 @@
   </head>
   <body>
     <h1>Trip Me Out! Fill me in more like!</h1>
+
+    <a href="pagevisit.html">Page Visit</a>
```

Comment:
Kinda strange you have this in both pulls. Did you branch one from the other or something?